playwright-java-junit/src/test/java/SampleTest.java

```diff
@@ -0,0 +1,50 @@
+import com.google.gson.JsonObject;
+import com.microsoft.playwright.Page;
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(DataProviderRunner.class)
+public class SampleTest extends  BaseTest{
+  @Test
+  @UseDataProvider(value = "getDefaultTestCapability",location = LTCapability.class)
+  public void sampleTest1(JsonObject capability){
+    Driver driver;
+    Page page = null;
+    try {
+      driver = super.createConnection(capability);
+      page = driver.getPage();
+      page.navigate("http://whatsmyuseragent.org/");
```

Comment:
Can we add the duckduckgo search example here as added here: https://github.com/LambdaTest/playwright-sample/blob/main/playwright-java/src/test/java/com/lambdatest/PlaywrightTestSingle.java
The example in the test is just navigating to a page.